conf/routes

```diff
@@ -13,6 +13,7 @@ GET     /AllFoodplacesWithOpenHours controllers.Application.getAllFoodplacesWith
 GET     /foodplaceOpenHours/:name   controllers.Application.getFoodplaceOpenHours(name: String)
 
 GET     /foodplaceVotes/:name       controllers.Application.getFoodplaceVotes(name: String)
+GET     /foodplaceVotes/chance/:name  controllers.Application.getOpenChanceFor(name: String)
```

Comment:
Why it's another route. I  can't really see a case when we want to use this request without previous one.